ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,169 @@
+  //
+  //  Copyright (c) Microsoft Corporation. All rights reserved.
+  //  Licensed under the MIT License.
+  //
+
+  import UIKit
+  import SwiftUI
+
+  ///Properties that make up cell content
+  @objc(MSFListVnextItem)
+  public class MSFListItem: NSObject, ObservableObject, Identifiable {
+      public var id = UUID()
+      @objc @Published public var leadingView: String?
+      @objc @Published public var title: String = ""
+      @objc @Published public var subtitle: String?
+  }
+
+  @objc(MSFListAccessoryVnextStyle)
+  /// Pre-defined styles of icons
+  public enum MSFListAccessoryVnextStyle: Int, CaseIterable {
+      case iconOnly
+      case withLabel
+      case disclosure
+  }
+
+  @objc(MSFListAccessoryVnextSize)
+  /// Pre-defined sizes of icons
+  public enum MSFListAccessoryVnextSize: Int, CaseIterable {
+      case icon
+      case persona
+      case disclosure
+      case button
+  }
```

Comment:
Why is this named size?
And what is the difference between disclosure here compared to the disclosure in MSFListAccessoryVnextStyle for example?